Makefile

```
@@ -65,12 +65,18 @@ fmt:
 vet:
 	go vet ./...
 
+# Run go lint against code
+check:
+	go run github.com/golangci/golangci-lint/cmd/golangci-lint run
+
+# Run meshery error utility against code
+error:
+	go run github.com/layer5io/meshkit/cmd/errorutil -d . update
```

```suggestion
	go run github.com/layer5io/meshkit/cmd/errorutil -d . update -i ./helpers -o ./helpers
```
Also move the errorutil files to the new directory

---

@DelusionalOptimist referenced a new issue for you to work on these changes.